.github/workflows/createReleaseBranch.yml

```diff
@@ -64,3 +64,15 @@ jobs:
     - run: echo “Release Type is ${{ github.event.inputs.releaseType || 'minor' }}”
     - run: echo “Release Branch is ${{ steps.branch.outputs.branch }}”
 
+  slack_notification:
+    if: ${{ always() }}
```

Comment:
Do you need the 'if' line here?

Reply:
heh..I asked the same q in the original PR.  From what I understand/remember it's required b/c it executes on success and failure.